src/anchor-click-interceptor.js

```diff
@@ -59,6 +60,14 @@ function maybeExpandUrlParams(ampdoc, e) {
     'CLICK_Y': () => {
       return e.pageY;
     },
+    '3PANALYTICS': (frameType, key) => {
+      const responses = ResponseMap.get(ampdoc, frameType,
```

Comment:
Copying conversation from old PR:

@keithwrightbos:
Why do you need responseMap? Idea was to use closure to keep track of data where message listener would update ampDoc.getAnchorClickListenerBinding() value.

Me:
I tried that, but it didn't seem like it was going to work. Though maybe I am missing something - I would be happy to pair on it if you'd like.
As it stands now, ResponseMap is now just a convenience wrapper around ampdoc's binding. I didn't want to spread the implementation details across both anchor-click-interceptor and amp-analytics, especially if they should ever change.

(We then discussed it offline some. Pairing is a possibility after Keith's travel, but as it's just an implementation barrier this should not derail getting it merged according to desired schedule.)